README.md

```diff
@@ -3,80 +3,101 @@
 [![Build Status](https://travis-ci.org/sourcecred/sourcecred.svg?branch=master)](https://travis-ci.org/sourcecred/sourcecred)
 [![Discord](https://img.shields.io/discord/453243919774253079.svg)](https://discord.gg/tsBTgc9)
 
-## Vision
-
-Open source software is amazing, and so are the creators and contributors who
-share it. How amazing? It's difficult to tell, since we don't have good tools
-for recognizing those people. Many amazing open-source contributors labor in
-the shadows, going unappreciated for the work they do.
-
-As the open economy develops, we need to go beyond [commit streaks] and
-follower counts. We need transparent, accurate, and fair tools for recognizing
-and rewarding open collaboration. SourceCred aims to do that.
-
-[commit streaks]: https://www.mxsasha.eu/blog/2016/04/01/how-github-contribution-graph-is-harmful/
-
-SourceCred will enable projects to create and track "cred", which is a
-quantitative measure of how much value different contributors added to a
-project. We'll do this by providing a basic data structure—a [cred graph]—into
-which projects can add all kinds of information about the contributions that
-compose it. For example, a software project might include information about
-GitHub pull requests, function declarations and implementations, design
-documents, community support, documentation, and so forth. We'll also provide
-an algorithm ([PageRank]) which will ingest all of this information and produce
-a "cred attribution", which assigns a cred value to each contribution, and thus
-to the people who authored the contributions.
-
-[cred graph]: https://en.wikipedia.org/wiki/Directed_graph
+SourceCred is an open-source system for creating reputation networks for open
+projects.  It does this by providing a Graph-based abstraction for tracking
+contributions and how contributions relate to each other, and [PageRank]
+inspired algorithms for attributing "cred" to contributions based on their
+importance.
+
+[PageRank]: https://en.wikipedia.org/wiki/PageRank
+
+#### An Example: SourceCred's cred
+
+SourceCred is initially focusing on supporting open-source projects. We think building
+tools for open-source projects to give public recognition to their contributors will
+help open-source sustainability.
+
+Naturally, we're dogfooding SourceCred on the SourceCred
+project itself. Most of our work happens on GitHub, so we import our GitHub data
+into a graph. Pulls and commits and users are all nodes, with edges connecting
+pulls to their authors, pulls to their commits, and so forth. Thus, if Pearl
+writes a bug report, Steven writes a pull request referencing that bug report, and
+Amethyst reviews and merges the pull, then Pearl, Steven, and Amethyst will all
+earn some cred in the project.
+
+Some contributions will be hard to spot on GitHub. For example, someone might
```

Comment:
Something about this paragraph rubs me the wrong way. I think that it’s
coming across as overly gamified and trivializing. What do you think of
rephrasing it to something like,

> Some contributions will be hard to spot on GitHub. For example,
> someone might help organize a SourceCred meetup, or work on community
> management. [To make sure those contributors get the cred they
> deserve, people can recognize them with spotlights.] The _spotlight_
> mechanism allows contributors to draw attention to contributions that
> might otherwise be overlooked. Multiple people can endorse a
> spotlight, increasing its value.

where the bracketed sentence could be removed with no complaint from me?

Reply:
Done